src/interfaces/Registration/business-lookup-interface.ts

```diff
@@ -0,0 +1,19 @@
+export interface BusinessLookupIF {
+  identifier: string
+  name: string
+  bn?: string
+}
+
+export const EmptyBusinessLookup: BusinessLookupIF = {
+  identifier: null,
+  name: null,
+  bn: null
+}
+
+export interface BusinessLookupResultIF {
+  identifier: string
+  legalType: string
+  bn: string
+  status: string
+  name: string
```

Comment:
Should legalType be of type CorpTypeCd?

Is there a type we can use for status?